drivers/watchdog/wdt_nrfx.c

```diff
@@ -93,6 +96,10 @@ static int wdt_nrf_install_timeout(const struct device *dev,
 	nrfx_err_t err_code;
 	nrfx_wdt_channel_id channel_id;
 
+	if (data->enabled) {
+		return -EBUSY;
```

Comment:
Please add error log here something like: `LOG_ERR("No more timeouts can be installed");` Also I think it should be `-ENOMEM`

Reply:
`-ENOMEM` is the documented return code:
https://github.com/zephyrproject-rtos/zephyr/blob/0f776cf0bfe85caed8281636354eb72454053624/include/zephyr/drivers/watchdog.h#L208-L209

Reply:
None of the other errors in this driver result in log messages, I am attempting to keep the changes as small as possible due to the v3.7 RC window.